core/server/worker/src/main/java/alluxio/worker/block/management/tier/AlignTask.java

```diff
@@ -143,6 +163,8 @@ public void run() {
           blockLocPairListDst.get(i).getFirst()));
     }
 
+    LOG.debug("Generated {} swap transfers: \n ->{}", transferInfos.size(),
```

Comment:
Do we need to wrap in `isDebugEnabled`? Also, based on our logging framework, will just using `debug` be sufficient to avoid the expensive portions for most of these?

Reply:
Yes, otherwise the parameters will be evaluated before making the decision to print.

Reply:
I also validated this on debugger.

Reply:
This message is not wrapped in `isDebugEnabled`?

Reply:
Oh. right.